src/puppetlabs/ezbake/core.clj

```diff
@@ -442,6 +442,25 @@ Additional uberjar dependencies:
             (get-timestamp-string))
     lein-version))
 
+(defn generate-package-version-from-version
+  [lein-version]
+  {:pre [(string? lein-version)]
+   :post [(string? %)]}
+  (if (.endsWith lein-version "-SNAPSHOT")
+    (format "%s"
+            (str/replace lein-version #"-SNAPSHOT|-" { "-SNAPSHOT" "" "-" "."}))
+    lein-version))
+
+(defn generate-package-release-from-version
+  [lein-version]
+  {:pre [(string? lein-version)]
+   :post [(string? %)]}
+  (if (.endsWith lein-version "-SNAPSHOT")
+    (format "%s.%s"
+            (str/replace lein-version #".*-" "0.1")
```

Comment:
for CPR-429, should we add the codename to the version or to the package name? I think in vanagon, it's appended on as a part of the release field (i.e., `puppet-agent_1.9.1-1xenial_i386.deb`), if that helps